cli/src/main/java/org/apache/falcon/cli/commands/FalconEntityCommands.java

```diff
@@ -18,9 +18,303 @@
+import static org.apache.falcon.cli.FalconEntityCLI.OFFSET_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.OFFSET_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.ORDER_BY_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.ORDER_BY_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.PATH_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.PATH_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.PROPS_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.PROPS_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.RESUME_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.RESUME_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.SCHEDULE_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.SCHEDULE_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.SHOWSCHEDULER_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.SHOWSCHEDULER_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.SKIPDRYRUN_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.SKIPDRYRUN_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.SLA_MISS_ALERT_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.SLA_MISS_ALERT_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.SORT_ORDER_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.SORT_ORDER_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.START_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.STATUS_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.STATUS_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.SUBMIT_AND_SCHEDULE_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.SUBMIT_AND_SCHEDULE_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.SUBMIT_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.SUBMIT_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.SUSPEND_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.SUSPEND_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.TAGKEYS_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.TAGKEYS_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.TAGS_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.TAGS_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.TYPE_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.UPDATE_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.UPDATE_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.VALIDATE_OPT;
+import static org.apache.falcon.cli.FalconEntityCLI.VALIDATE_OPT_DESCRIPTION;
+import static org.apache.falcon.cli.FalconEntityCLI.validateEntityFields;
+
 /**
  * Entity Commands.
  */
+// SUSPEND CHECKSTYLE CHECK ParameterNumberCheck
```

Comment:
Instead of suspending check for entire class, how about moving it to specific methods?